paper.md

```diff
@@ -45,51 +45,51 @@ bibliography: paper.bib
 It provides a general approach for complex problems in two or three dimensions and offers simulation results of high accuracy and depth.
-The main disadvantage is the high computational effort in solving the equation systems, increasing rapidly with non-linearity of the model equations and the resolution of the solution space.
-Therefore, this method has grown along with the development of high performance computer systems in the past decades.
-Usage of FEM increases the achievable modelling depth immensely at the expense of high calculation times.
+The main disadvantage is the high computational effort in solving the equation systems, increasing rapidly with the non-linearity of the model equations and the resolution of the solution space.
+Therefore, this method has grown along with the development of high-performance computer systems in the past decades.
+The usage of FEM increases the achievable modelling depth immensely at the expense of high calculation times.
 However, none of the authors cited provides the source code of the simulation programs used or the corresponding build files for the used commercial FEM toolkits.
-This circumstance makes it difficult for other authors to compare own results with results from literature as they are unable to reproduce the results without extensive efforts.
+This circumstance makes it difficult for other authors to compare their own results with results from the literature, as they are unable to reproduce the results without extensive effort.
 
-In the field of metal forming simulation in general, the authors of this software know only of very few authors providing their source code, namely @Alexander1972 and @Pawelski2000.
-The field of groove rolling simulation is, according to the authors' experience, characterized by usage of handcrafted, specialized, not reusable software tools, or, by usage of large commercial packages.
+**(Comment): This statement seems a bit misleading; for example, many open-source finite element (or similar) codes are capable of metal forming simulation, i.e. contact with large plastic strains, e.g. [CalculiX](https://calculix.discourse.group/t/contact-convergence-in-metal-forming-simulation/1357), [FEBio](https://febio.org/knowledgebase/case-studies/structural-mechanics/reactive-plasticity-benchmark-problem/), [deal.II](https://www.dealii.org/current/doxygen/deal.II/step_42.html), my own [solids4foam](https://www.solids4foam.com/tutorials/more-tutorials/solid-mechanics/elastoplasticity/cooksMembrane.html) OpenFOAM toolbox ([paper](https://onlinelibrary.wiley.com/doi/abs/10.1002/nme.5345)).** In the field of metal forming simulation in general, the authors of this software know only of very few authors providing their source code, namely @Alexander1972 and @Pawelski2000.
```

Comment:
Hi @ChRen95, to be honest, from reading the paper and the documentation, I struggled to understand what approach was adopted. So, yes, I think it would be a good idea to clarify the focus of this approach on analytical solutions and make it clear within the paper and documentation that this is the approach that it adopted.